dev/bin/logs-shard.sh

```diff
@@ -0,0 +1,25 @@
+#!/bin/bash -e
+
+########
+# Read the shard operator logs
+########
+
+. $(dirname "${BASH_SOURCE[0]}")/common.sh
+
+configure_cluster_started
+
+if [ "${DEV_ENVIRONMENT}" = 'true' ]; then
+    tail -f ${dev_logs_dir}/shard-operator.log
+else
+    pod_name=$(oc get pods -n event-bridge-operator | grep event-bridge-shard-operator | grep Running | awk '{print $1}' | head -n 1)
+    if [ -z ${pod_name} ]; then
+        echo 'Cannot find the manager pod...'
+        echo
+        oc get pods -n event-bridge-operator
```

Comment:
Please convert to kubectl or add oc to required tools

Reply:
woops ...